src/Filament/Actions/MediaAction.php

```diff
@@ -70,7 +82,7 @@ protected function setUp(): void
                     type: 'media',
                     statePath: $component->getStatePath(),
                     media: [
-                        'src' => $source . '',
+                        'src' => $source,
                         'alt' => $data['mediaModel']->alt_text ?? null,
```

Comment:
What is `title` used for? Also, why is there error checking on `alt` but not on `title`?

Reply:
Good call out.  This is ridiculous because we don't even have a `title` field on the table.  I'll remove.